[BUG] Error (not affecting dataset download) when running container

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Expected Behavior

Dataset install + download without error.

### Current Behavior

When running a `dataget` Docker container, I get an error message before the specified datasets are installed/downloaded. The error message does not appear to prevent the dataset downloading process.

### Error message

```bash
Will now install DatasetID from PortalURI.
[ERROR] NoDatasetFound(No dataset found at '/data/output' for the purpose "get content of ['PortalURI']".  Specify a dataset to work with by providing its path via the `dataset` option, or change the current working directory to be in a dataset.) (NoDatasetFound)
usage: datalad install [-h] [-s URL-OR-PATH] [-d DATASET] [-g] [-D DESCRIPTION] [-r] [-R LEVELS]
                       [--reckless [auto|ephemeral|shared-...]] [-J NJOBS] [--branch BRANCH] [--version]
                       [URL-OR-PATH ...]
[ERROR] NoDatasetFound(No installed dataset found at /data/output/SessionPath) (NoDatasetFound)
usage: datalad get [-h] [-s LABEL] [-d PATH] [-r] [-R LEVELS] [-n]
                   [-D DESCRIPTION] [--reckless [auto|ephemeral|shared-...]]
                   [-J NJOBS] [--version]
                   [PATH ...]
```

### Environment

- OS: Ubuntu 22.04.1 LTS

### How to reproduce

On https://query.neurobagel.org, submit a query for males between the ages of 15-30, with diagnosis "Major depressive disorder". Download the TSVs for the third resulting dataset "Neural Processing of Emotional Musical a...".

Then, in the directory containing the TSVs, run 
```bash
docker run -t -v $(pwd):/data neurobagel/dataget:latest /data/dataset-level-results.tsv /data/participant-level-results.tsv /data/output
```

### Anything else?

Based on the error message it seems that the header line of the `dataset-level-results.tsv` is not being skipped properly during datalad parsing.

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -13,6 +13,8 @@ export -f getit
 [ ! -e ${out} ] && mkdir -p ${out}
 
 {
+    # We need to read the first line and discard it to skip the header row
+    read
     # Ensure last line is read even if file does not end with newline
     # See: https://stackoverflow.com/a/12916758
     while read -r dataset || [ -n "$dataset" ];

